ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -12966,6 +12969,23 @@ keys:
             min_length: 1
             items:
               type: str
+          bindings_timeout:
+            type: int
+            description: Timeout for bindings stored on STUN server in seconds
+            convert_types:
+            - str
+            min: 10
+            max: 7200
+          ssl_profile:
+            type: str
+            description: SSL profile name
+          ssl_connection_lifetime:
```

Comment:
I'm unsure whether this is the right way to model this. This can be both in minutes and hours, so for now I just added support for minutes and avoided adding support for the hour format. Is this the right way to do this?